app/controllers/api/v2/job_templates_controller.rb

```diff
@@ -22,6 +22,23 @@ def index
         @job_templates = resource_scope_for_index
       end
 
+      api :POST, '/job_templates/import', N_('Import a job template from ERB')
+      param :template, String, :required => true, :desc => N_('Template ERB')
+      param :overwrite, :bool, :required => false, :desc => N_('Overwrite template if it already exists')
+      def import
+        options = params[:overwrite] ? { :update => true } : { :build_new => true }
+
+        @job_template = JobTemplate.import(params[:template], options)
+        @job_template ||= JobTemplate.new
+        process_response @job_template.save
+      end
+
+      api :GET, '/job_templates/:id/export', N_('Export a job template to ERB')
+      param :id, :identifier, :required => true
+      def export
+        send_data @job_template.to_erb, :type => 'text/plain', :disposition => 'attachment', :filename => @job_template.filename
```

Comment:
`@template.filename + '.txt'` helps my browser to get "a.erb.txt" instead of "a.bin" with unknown extension, not entirely sure why, headers seems to be correct in both cases